ramls/group-fund-fiscal-year.raml

```diff
@@ -0,0 +1,75 @@
+#%RAML 1.0
+
+title: Finance group fund fiscal year
+version: v1
+protocols: [ HTTP, HTTPS ]
+baseUri: https://github.com/folio-org/mod-finance
+
+documentation:
+  - title: Group Fund Fiscal Year API
+    content: This documents the API calls that can be made to manage group-fund-fiscal-years
+
+types:
+  groupFundFiscalYear: !include acq-models/mod-finance/schemas/group_fund_fiscal_year.json
+  groupFundFiscalYearCollection: !include acq-models/mod-finance/schemas/group_fund_fiscal_year_collection.json
+  errors: !include raml-util/schemas/errors.schema
+  UUID:
+    type: string
+    pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
+
+traits:
+  validate: !include raml-util/traits/validation.raml
+  language: !include raml-util/traits/language.raml
+  pageable:  !include raml-util/traits/pageable.raml
+  searchable: !include raml-util/traits/searchable.raml
+
+resourceTypes:
+  collection: !include raml-util/rtypes/collection.raml
+  collection-item: !include raml-util/rtypes/item-collection.raml
+
+/finance/group-fund-fiscal-years:
+  displayName: Finance group fund fiscal year
+  description: Finance group fund fiscal year APIs
+  type:
+    collection:
+      schemaCollection: groupFundFiscalYearCollection
+      schemaItem: groupFundFiscalYear
+      exampleCollection: !include acq-models/mod-finance/examples/group_fund_fiscal_year_collection.sample
+      exampleItem: !include acq-models/mod-finance/examples/group_fund_fiscal_year.sample
+  post:
+    is: [validate]
+  get:
+    is: [
+      pageable,
+      searchable: {description: "with valid searchable fields: for example fundId", example: "[\"fundId\", \"6e2fbba3-d557-4480-bca3-b6f5c645de04\", \"=\"]"}
+    ]
+  /{id}:
+    uriParameters:
+      id:
+        description: The UUID of a group fund fiscal year
+        type: UUID
+    delete:
+      is: [validate, language]
+      displayName: Budget
+      description: Delete a group fund fiscal year
+      responses:
+        204:
+         description: "Item deleted successfully"
+        404:
+          description: "Item with a given ID not found"
+          body:
+            text/plain:
```

Comment:
Th BL logic is designed to return `application/json` wherever possible. Please declare this type as well.
Ideally, [mod-orders rtypes](https://github.com/folio-org/mod-orders/tree/master/ramls/rtypes) can be moved to common place (e.g. to acq-models repo) and used by BL modules because at the moment contract (defined by raml file) isn't followed by implementation.
@folio-org/acquisitions I would like to get your thoughts about it

Reply:
Story [MODFIN-64](https://issues.folio.org/browse/MODFIN-64) created.